ci/publish-artifactory.sh

```diff
@@ -51,11 +51,14 @@ for base in non-repudiation-core non-repudiation-client; do
     done
 done
 
-for platform in linux macos windows; do
-    EE_TARBALL=daml-sdk-$RELEASE_TAG-$platform-ee.tar.gz
-    push sdk-ee $EE_TARBALL
-    push sdk-ee $EE_TARBALL.asc
-done
-EE_INSTALLER=daml-sdk-$RELEASE_TAG-windows-ee.exe
-push sdk-ee $EE_INSTALLER
-push sdk-ee $EE_INSTALLER.asc
+# For the split release process these are not published to artifactory.
+if [[ "$#" -lt 3 || $3 != "split" ]]; then
```

Comment:
this is a bit ugly but given that I’m hoping we can just drop the non-split release process in a few weeks and just delete this, this seems better than duplicating the whole file.

Reply:
Since YAML files are from main, will we still be able to do patch releases if we drop the non-split process?

Reply:
I’m not following, this is not a yaml file it will never be used for old releases.

Reply:
oh sorry, I think I was just suggesting we can drop it from this file not from the yaml files. Does that make more sense?

Reply:
Yes, that does. 👍